### PR TITLE
Add e2e test for draft provider and provider created by fully filled form

### PR DESCRIPTION
**Pull request by Marcelinna**
Add e2e test for draft provider and provider created by fully filled form
Add docker compose file for e2e and cypress-grep dependency

### CODE REVIEW

File: test/system/package.json [hunk 1]
```diff
@@ -4,10 +4,10 @@
   "description": "Integration tests",
   "main": "",
   "scripts": {
-    "cy:run": "TZ=Europe/Warsaw cypress run --browser chrome --headless --quiet",
+    "cy:run": "TZ=Europe/Warsaw cypress run --browser chrome --headless --quiet --env grepTags='-@extendend-test'",
```
Comment (jswk) [inline test/system/package.json:7]: A typo: `extendend` should instead read `extended`. Please correct it so it doesn't propagate.

**Review by jswk — APPROVED** [review]
Apart from the typo in the tag it's good to go.

**Review by kmarszalek — APPROVED** [review]

File: test/system/cypress/integration/regression_tests/backoffice/providers/providers.spec.ts [hunk 2]
```diff
@@ -148,11 +148,7 @@ describe("Providers", () => {
       .then((value) => {
         cy.contains("a", "Delete")
           .click();
-<<<<<<< HEAD
-        cy.contains("div.alert-success", "Provider removed successfully")
-=======
         cy.contains("div.alert-success", successDeletionMessage)
->>>>>>> Add e2e tests for draft and created by fully form providers
```
Comment (jswk) [inline test/system/cypress/integration/regression_tests/backoffice/providers/providers.spec.ts]: Don't commit unresolved conflicts, even if they are to be removed in a subsequent commit.

**Review by jswk — CHANGES_REQUESTED** [review]
Please remove the unresolved conflicts from the first commit.

**Review by jswk — APPROVED** [review]

**Pull request merged**